geode-core/src/main/java/org/apache/geode/internal/cache/DistributedRegion.java

```diff
@@ -181,6 +182,7 @@
    * Lock to prevent multiple threads on this member from performing a clear at the same time.
    */
   private final Object clearLock = new Object();
+  private final ReentrantReadWriteLock failedGIIClearLock = new ReentrantReadWriteLock(true);
```

Comment:
a better name might be "failedInitialImageLock". We want to get away from acronyms. And "clear" does not need to be in the name.